lib/dish.rb

```diff
@@ -0,0 +1,12 @@
+class Dish
+
+  attr_reader :name, :price, :item
+  def initialize(name, price)
+    @price = price
+    @name = name
+  end
+
```

Comment:
Whats the difference between the method created using `attr_reader :item` and the one created using `def item`?

Do you need both?

Reply:
I've removed the attr_reader one as I'm setting up ` @item` to an hash. Then the tests are still passing.